### PR TITLE
Fixes from qa-09 with changelog entry and wip clear

### PR DESCRIPTION
**Pull request by OlleLarsson**
**What this PR does / why we need it**:
This pr includes cherry-picked fixup commits from the v0.9.0 release.
I did not include the actual v0.9.0 release commit as i wasn't sure if we want that on main or not.

Added commits that adds the v0.9.0 entry to the changelog and clear the wip-changelog.

**Checklist:**

- [ ] Added relevant notes to [WIP-CHANGELOG.md](https://github.com/elastisys/compliantkubernetes-apps/blob/main/WIP-CHANGELOG.md)
- [ ] Proper commit message prefix on all commits
- [ ] Updated the [public facing documentation](https://github.com/elastisys/compliantkubernetes)
- Is this changeset backwards compatible for existing clusters? Applying:
    - [ ] is completely transparent, will not impact the workload in any way.
    - [ ] requires running a migration script.
    - [ ] will create noticeable cluster degradation.
          E.g. logs or metrics are not being collected or Kubernetes API server
          will not be responding while upgrading.
    - [ ] requires draining and/or replacing nodes.
    - [ ] will change any APIs.
          E.g. removes or changes any CK8S config options or Kubernetes APIs.
    - [ ] will break the cluster.
          I.e. full cluster migration is required.

### CODE REVIEW

**Review by viktor-f — APPROVED** [review]
LGTM

**Review by Xartos — APPROVED** [review]
LGTM!

File: WIP-CHANGELOG.md [hunk 1]
```diff
@@ -1,80 +1,10 @@
...
-- StorageClasses are now enabled/disabled in the `{wc,sc}-cofig.yaml` files.
-- Mount path and IP/hostname is now configurable in `nfs-client-provisioner`.
-- Upgraded `cert-manager` to `1.1.0`.
-- Moved the `bootstrap/letsencrypt` helm chart to the apps step and renamed it to `issuers`.
-  The issuers are now installed after cert-manager.
-  You can now select which namespaces to install the letsencrypt issuers.
-- Helm upgraded to `v3.5.0`.
-- InfluxDB upgraded to `v4.8.12`.
-- Resource requests/limits have been updated for all Harbor pods.
 - The pipeline now stops on lint errors.
```
Comment (OlleLarsson) [inline WIP-CHANGELOG.md:6]: This should be remove as it is about the pipeline, right?

Comment (lentzi90) [reply WIP-CHANGELOG.md:6]: Well it is about the scripts in this repo, but it isn't a user facing change so I don't think it is needed here. Fine to remove. :slightly_smiling_face:

**Review by lentzi90 — APPROVED** [review]
LGTM!

**Pull request merged**